cloudflared/DOCS.md

```diff
@@ -64,10 +64,12 @@ Example extended add-on configuration:
 external_hostname: "ha.example.com"
 tunnel_name: "homeassistant"
 additional_hosts:
+  - hostname: "router.example.com"
+    service: "http://192.168.1.1"
   - hostname: "diskstation.example.com"
-    service: "http://192.168.1.5"
-  - hostname: "website.example.com"
     service: "http://192.168.1.2"
```

Comment:
I think this is great, but we could go a step further and also show that https is working. 
So my idea would be to change this to "https://192.168.1.2:5001"

Reply:
Is it good now?